### DISCUSSION TITLE
What is `git`?

### DISCUSSION THREAD

**Discussion by xlgmokha** [CodeChica/plus-plus#33] [General]
I have so many questions. Like...

* What is a version control system?
* What is this `git` thingy?
* Why do computer programmers use `git`?

Please explain like I'm 5 years old.

**Comment by daitanfutekina** [comment]
1) system that keeps track of changes and is overarching of git, it also protects versions.
2) it keeps track of changes In files its different from others since its ✧･ﾟ: *✧･ﾟ:*FREE *:･ﾟ✧*:･ﾟ✧.
3) it has the advance of being free but it also has an environment for an online workspace since it allows multiple people to see the same data and information.
